common/lib/xblock/__init__.py

```diff
@@ -0,0 +1,10 @@
+'''
```

Comment:
Can you try moving this to ``openedx/core/lib``, as we are trying to deprecate the common directory (or at least not add any new code to it).